zipline/examples/.gitignore

```diff
@@ -0,0 +1 @@
+.ipynb_checkpoints
```

Comment:
@ssanderson Should this be added to the top level `.gitignore` file rather than adding another `.gitignore`?